.github/workflows/build.yaml

```diff
@@ -19,12 +23,42 @@ jobs:
         with:
           install_url: https://releases.nixos.org/nix/nix-2.22.1/install
 
+      - name: Use GitHub cache
+        uses: DeterminateSystems/magic-nix-cache-action@main
+
       - name: Check Nix formatting
         run: nix fmt . -- --check
 
       - name: Check Nix flake
         run: nix flake check --all-systems
 
+  script-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: azohra/shell-linter@latest
+
+      - name: Test bump-holochain.sh
+        run: |
+          MINOR_VERSION=$(./scripts/bump-holochain.sh main-0.1 false | tail -n 1)
```

Comment:
Same reason as above, I just need a set of tags to work with. I could find another way to test it like getting the tag list from a file rather than git but this doesn't require any fiddles that might break.

It will break one day if we change repo or remove old tags or something, but for now this is unlikely to break